code/datums/abilities/critter/drop_disguise.dm

```diff
@@ -0,0 +1,26 @@
+/datum/targetable/critter/drop_disguise
+	name = "Drop Disguise"
+	desc = "Shed your skin and return to your base form."
+	icon_state = "drop_disguise"
+	cooldown = 3 SECONDS
+	targeted = 0
+
+	cast()
+		if (..())
+			return TRUE
+		var/mob/living/critter/mimic/user = holder.owner
+		if (user.base_form)
+			boutput(holder.owner, SPAN_ALERT("You're in your base form already!"))
+			return TRUE
+		SETUP_GENERIC_PRIVATE_ACTIONBAR(user, null, 2 SECONDS, /datum/targetable/critter/drop_disguise/proc/drop, user, null, null, null, INTERRUPT_MOVE | INTERRUPT_ACT | INTERRUPT_ACTION)
+		boutput(holder.owner, SPAN_ALERT("You begin to shed your skin..."))
+		return FALSE
+
+	proc/drop(mob/user)
+		var/mob/living/critter/mimic/parent = user
+		var/datum/targetable/critter/mimic/abil = parent.getAbility(/datum/targetable/critter/mimic)
+		abil.afterAction()
```

Comment:
Honestly this just feels unnecessary, you've watched the timer count down and then it gets reset when you drop your disguise anyway. Rah!